src/KafkaFlow/Clusters/ClusterManager.cs

```diff
@@ -75,5 +76,12 @@ public async Task CreateIfNotExistsAsync(IEnumerable<TopicConfiguration> configu
                 }
             }
         }
+
+        private class DuplicateTopicConfigurationEqualityComparer : IEqualityComparer<TopicConfiguration>
+        {
+            public bool Equals(TopicConfiguration x, TopicConfiguration y) => x?.Name == y?.Name;
+
+            public int GetHashCode(TopicConfiguration obj) => obj.Name.GetHashCode();
+        }
```

Comment:
Maybe throwing an exception should be better. What do you think?

Reply:
Since KafkaFlow doesn't throw an exception if it already exists, I think it makes sense not to throw it in case of duplicated configuration. I believe that as a consumer, I would expect the first attempt to succeed and the second to be ignored.

Reply:
Hi @kYann 
We discussed this issue and @filipeesch raised a scenario that we are concerned with.
If eventually, we have a duplicate topic declaration with different parameters (number of partitions and replicas), a silent deduplication process may induce confusion.
So, we believe that at least we should log a warning mentioning the duplication and that we are deduplicating and using the settings X and Y to try to create the topic.

What do you think?

Reply:
Good point. I think the warning is a good idea. There is no other way to resolve this.
If the warning could happen only when the parameters are different would be a nice touch.